src/progressbar/progressbar.ts

```diff
@@ -38,10 +37,12 @@ export class NgbProgressbar {
 
   @Input()
   set value(value: number) {
```

Comment:
I think we still need them for max as currently we don't handle max-as-string properly